src/Our.Umbraco.Tree.Config/LoadConfigFiles.cs

```diff
@@ -71,8 +73,29 @@ public override void Render(ref XmlTree tree)
 			    }
 			    xNode.OpenIcon = xNode.Icon;
 				tree.Add(xNode);
-			}
-		}
+
+                // Add extra config files specified in ExtraConfigFiles setting.
+                var items = ConfigurationManager.AppSettings["ExtraConfigFiles"].Split(',').Select(s => new { Title = s.Split(':').First(), Filename = s.Split(':', '=').Last() });

```

Comment:
I'm not sure about the Title/Filename format (for the AppSetting value) yet.

My suggestion would be to add the file-paths in the AppSetting value, (comma-separated), then on the tree it would display the path + filename.

@TimGeyssens any thoughts about this?